src/components/navigators/BottomNavigator/BottomNavigator.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+import { Home, HomeFill, Exploration, ExplorationFill, Diary, DiaryFill } from "@/components/icons";
+import * as S from "./BottomNavigator.style";
+import { BOTTOM_NAV_INABLE } from "../navInablePath";
+
+interface BottomNavItemProps {
+  path: string;
+  icon?: JSX.Element;
+  activeIcon?: JSX.Element;
```

Comment:
[ P3 ] 제가 테스트할 수 있는 환경이 안돼서 정상 동작할지 모르겠지만 ㅠ  된다면! 
아래 처럼 수정해 주신다면 컨벤션에도 잘 맞을것 같아요!  

```suggestion
  icon?: React.ReactNode;
  activeIcon?: React.ReactNode;
```